helm/defectdojo/values.yaml

```diff
@@ -495,6 +495,16 @@ redis:
 #   DD_SOCIAL_AUTH_AUTH0_SECRET: 'xxx'
 extraConfigs: {}
 
+# Extra fixtures can be mounted to the initializer job by adding an item to the extraFixtures list:
+# NOTE  Each fixture must have a 'data' section, which must include a 'model' string and a 'fields' map
+# NOTE  These fixtures are only applied to an uninitialized database
+# extraFixtures:
+# - data:
+#   - model: dojo.system_settings
+#     fields:
+#       enable_deduplication: true
+extraFixtures: []
```

Comment:
Can you provide example how extraFixuters should look like?

Reply:
Is this the usecase? If it is, you should use api to bootstrap your app.

Reply:
It's the simplest usecase I could think of, but users could use this to pre-populate their databases with anything from additional test types to custom surveys

Reply:
The API would be better for most post-install configurations like this. I may go that direction for what I initially made this for, but this feature is still potentially useful. Whether it's useful enough to maintain is up to the you all